components/notebook-controller/go.mod

```diff
@@ -3,74 +3,74 @@ module github.com/kubeflow/kubeflow/components/notebook-controller
 go 1.20
```

Comment:
openshift-ci is not happy

```
[1/2] STEP 8/8: RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -a -mod=mod -o manager main.go
# sigs.k8s.io/controller-runtime/pkg/cache
/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.17.0/pkg/cache/cache.go:424:58: fields.Selector does not implement comparable
/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.17.0/pkg/cache/cache.go:454:20: fields.Selector does not implement comparable
note: module requires Go 1.21
```

Reply:
I see, I open this PR on ocp-ci to get aligned. https://github.com/openshift/release/pull/52888

Reply:
/retest

Reply:
/retest